torch/lib/THD/master_worker/common/Functions.hpp

```diff
@@ -208,7 +208,9 @@ enum Functions: std::uint16_t {
   tensorLogspace,
   tensorRand,
   tensorRandn,
+  tensorLogicalallall,
```

Comment:
The changes in this part of THD aren't necessary. We'll be refactoring it to automatically generate this code. Still you don't have to revert these changes, just keep that in mind for the future

Reply:
Thanks for your clarification. I also rename `logicalall` to `logicalAnd` and `logicalallall` to `logicalAndAll` for consistence.